mnefun/_sss.py

```diff
@@ -577,16 +577,27 @@ def _pos_valid(pos, dist_limit, gof_limit):
 def _get_t_window(p, raw):
     t_window = p.coil_t_window if p is not None else 'auto'
     if t_window == 'auto':
-        from mne.chpi import _get_hpi_info
         try:
             hpi_freqs, _, _ = _get_hpi_info(raw.info, verbose=False)
+            info = raw.info
+            if info['line_freq'] is not None:  # as in mne.chpi.py
+                line_freqs = np.arange(info['line_freq'], info['sfreq'] / 3.,
+                                       info['line_freq'])
+            else:
+                line_freqs = np.array([60])   # SMB: should be safe to assume
```

Comment:
```suggestion
            info = raw.info
            line_freq = info['line_freq'] or 60.
            line_freqs = np.arange(line_freq, info['sfreq'] / 3., line_freq)
```
the `or` short-circuit trick here should be safe enough for us